src/utils/prepare-file.js

```diff
@@ -77,30 +84,79 @@ function prepareFile (file, opts) {
   let files = [].concat(file)
 
   return flatmap(files, (file) => {
-    if (typeof file === 'string') {
-      if (!isNode) {
-        throw new Error('Can only add file paths in node')
-      }
+    return prepare(file, opts)
+  })
+}
 
-      return loadPaths(opts, file)
-    }
+function prepare (file, opts) {
+  // probably it should be valid and would be handled below with Buffer.from
+  if (typeof file === 'string') {
+    throw new Error('String isn\'t valid as an input')
```

Comment:
Use [err-code](https://www.npmjs.com/package/err-code) here. We are trying to use errors with code everywhere.